src/SuperVault.sol

```diff
@@ -186,17 +186,15 @@ contract SuperVault is BaseStrategy, ISuperVault {
         emit DepositLimitSet(depositLimit_);
     }
 
-    /// @notice Sets the strategist for the vault
-    /// @param strategist_ The new strategist
-    function setStrategist(address strategist_) external onlyManagement {
+    /// @inheritdoc ISuperVault
+    function setStrategist(address strategist_) external override onlyManagement {
```

Comment:
All functions declared in interfaces are implicitly virtual and any functions that override them do not need the override keyword.